### DISCUSSION TITLE
Do you have a plan?

### DISCUSSION THREAD

**Discussion by ghost** [quasarframework/quasar#17622] [General - Components / Directives / etc] [answered]
Hi, Team

Do you have a plan for version upgrade?
Quasar is not develop like past.

Thaks

**Comment by rstoenescu** [comment, marked as answer]
Hi,

Yes, we have a plan. We would have released the new q/app-s (I think that you are referring to them) last week if I didn't get a breakthrough very recently for the BEX mode (HMR for Chrome! is one of them).

This is always the case when we are taking a big leap forward (UI or CLI) and lots of research is needed. The development appears to slow down. However, in reality it's quite the opposite. Every. Time. The new CLIs are such a major improvement and the work we are putting into them was and is so insane that you, the community, would not believe it. Like with the previous major releases, we want to ship the best product ever, and it takes time. We also don't want to get "aaah, but this fix will make a breaking change" type of moments (and then release another major version). CLI development is hard. We're doing things no one else has. The ESM changes have huge implications and that's why testing & feedback from the community is really important. What worsened the situation for the new CLIs is that we started with a plan that kept on changing due to the ecosystem (the rapid release of Vite 3, 4 & 5 is one of the reasons).

In any case, this week we will release the final beta (take it as an RC too) for both the CLIs and then things will get back to what you were used to before.

PS: For those working at companies using Quasar, we would really appreciate if you could ask your employers to sponsor/donate to Quasar. It would mean quite a lot for us and it would speed up the development. https://donate.quasar.dev
